packages/babel-cli/src/babel/options.js

```diff
@@ -28,6 +28,11 @@ commander.option(
   "The name of the 'env' to use when loading configs and plugins. " +
     "Defaults to the value of BABEL_ENV, or else NODE_ENV, or else 'development'.",
 );
+commander.option(
+  "--root-mode [mode]",
+  "The project-root resolution mode. " +
+    "One of 'root' (the default), 'upward', or 'upward-optional'.",
```

Comment:
I think that Instead of `root` could we name it `cwd`? or `.`?

Reply:
`cwd` and `root` can be two different things when running Babel, so it would be confusing.

Reply:
I'm not sure mentioning the working directory in this message will help honestly. Without the docs, I doubt users would know what any of these 3 mean anyway.